backend/routes/event.js

```diff
@@ -0,0 +1,95 @@
+const express = require("express");
+const router = express.Router();
+const auth = require("../util/auth");
+
+const Event = require("../models/event.model");
+
+router.get("/", (req, res, next) => {
```

Comment:
We should be using `async` for this.

Reply:
this function is to be async right, so what's happening here is I am passing another function in arrow syntax as a callback, another way of writing async-await!

Reply:
Alright, seems good to me then!